autovacuum_message_attachment/models/vacuum_rule.py

```diff
@@ -1,7 +1,7 @@
 # Copyright (C) 2018 Akretion
 # License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
 
-from odoo import _, api, exceptions, fields, models
+from odoo import api, exceptions, fields, models
```

Comment:
Please move the method  _compute_model_id(self)  under the fields to be coherent with current format.
And maybe invert the default by putting it on top of the condition scope and maybe avoid the codecov issue.